opentelemetry-sdk/src/opentelemetry/sdk/util.py

```diff
@@ -26,6 +28,31 @@
     from collections import Sequence
 
 
+def set_timeout_signal_handler():
+    "Signal timeout setter."
+
+    def signal_handler_function(signum, frame):
```

Comment:
This is a function, there is no need to add `_function` to its name. The name of the function should describe what the function does, in this case, it raises a `TimeoutError`.